rebasehelper/utils.py

```diff
@@ -817,15 +817,17 @@ class GitHelper(object):
     @classmethod
     def get_user(cls):
         try:
-            return git.cmd.Git().config('--global', 'user.name', get=True, stdout_as_string=six.PY3)
+            return git.cmd.Git().config('user.name', get=True, stdout_as_string=six.PY3)
         except git.GitCommandError:
+            logger.warning("Faild to get Git user.name. Return the fallback value '%s'", cls.GIT_USER_NAME)
```

Comment:
Ok, doesn't this sound better? `"Failed to get configured git user name, using '%s'"`

Reply:
Your way looks better and more simple. I will fix it.